### PR TITLE
Disable CA rotation by default for now

### PR DESCRIPTION
**Pull request by nightkr**
# Description

https://github.com/stackabletech/issues/issues/514 showed that we end up generating keystores that not all products currently support.

As a workaround we're putting rotation behind a feature flag until that has been resolved.

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
# Author
- [x] Changes are OpenShift compatible
- [ ] CRD changes approved
- [x] CRD documentation for all fields, following the [style guide](https://docs.stackable.tech/home/nightly/contributor/style-guide).
- [x] Helm chart can be installed and deployed operator works
- [x] Changes need to be "offline" compatible
```

```[tasklist]
# Reviewer
- [x] Code contains useful comments
- [ ] (Integration-)Test cases added
- [x] Documentation added or updated. Follows the [style guide](https://docs.stackable.tech/home/nightly/contributor/style-guide).
- [x] Changelog updated
- [x] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
```

### CODE REVIEW

File: docs/modules/secret-operator/pages/secretclass.adoc
Comment (sbernauer) [inline docs/modules/secret-operator/pages/secretclass.adoc:69]:
```suggestion
and can be configured to initiate rotation once less than half of that time remains (by setting `autoTls.ca.experimentalRotateCaCertificate` to `true`).
```

**Review by sbernauer — APPROVED** [review]
LGTM, only minor doc comment. Do we want to wait for an CRD approval? I would also be happy to optimistically merge and present the change in Arch meeting

**Comment by nightkr** [comment]
Given that it's a blocker, I'd try to get it in asap.

**Comment by soenkeliebau** [comment]
I agree, lets not wait for the arch meeting.

**Review by sbernauer — APPROVED** [review]

**Pull request merged**
